react/features/calendar-sync/components/BaseCalendarList.js

```diff
@@ -174,6 +176,20 @@ class BaseCalendarList extends Component<Props> {
         this.props.dispatch(refreshCalendar(true));
     }
 
+    _onSecondaryAction: string => void;
+
+    /**
+     * Handles the list's secondary action.
+     *
+     * @private
+     * @param {string} id - The ID of the item on which the secondary action was
+     * performed.
+     * @returns {void}
+     */
+    _onSecondaryAction(id) {
+        this.props.dispatch(maybeUpdateCalendarEvent(id, ''));
```

Comment:
I'm thinking this could be renamed to something like "openAddCalendarEventConfirmationDialogForPrivacyReasonsLongName" to be explicit about what it does, unless you need to abstract the name.

Reply:
Be careful what you wish for, I almost done it.

Now it's ```openUpdateCalendarEventDialog```, looks better to me too. However please note that this is a passthrough call for web until the web confirm dialog is implemented